src/NewProject/ProjectManager/project_manager.cpp

```diff
@@ -62,6 +66,39 @@ void ProjectManager::CreateProject(const ProjectOptions& opt) {
   setTopModule(opt.topModule);
   setTopModuleLibrary(opt.topModuleLib);
 
+  std::string delimiter = " ";
+  // Set Library Paths
+  if (!opt.libraryPathList.isEmpty()) {
+    std::vector<std::string> tokens;
+    StringUtils::tokenize(opt.libraryPathList.toStdString(), delimiter, tokens);
+    for (auto file : tokens) {
+      std::string cmd = "add_library_path " + file;
+      GlobalSession->CmdStack()->push_and_exec(new Command(cmd));
```

Comment:
I'm not sure about this. This is not like it done for all other fields. But it make sense because you don't need to copy logic inside tcl commands.

Reply:
Yeah i wasn't sure either, the original commit on the pr does it the more standard way, but I noticed that the add_library_path tcl code has special path logic i hadn't capture so i tried invoking via tcl to get the same results. I used https://github.com/os-fpga/FOEDAG/blob/477b70d0d638ee0fb1b8a2c45ac08d950258dbae/src/MainWindow/main_window.cpp#L482 as an example I think this might actually be how we are supposed to be updating the _cmd.log as the user drives the UI, but that's a discussion for later.

Reply:
yes, probably